crates/swc_ecma_codegen/src/tests.rs

```diff
@@ -872,17 +872,17 @@ fn ascii_only_tpl_lit() {
 #[test]
 fn ascii_only_issue_7240() {
     test_all(
-        r"
+        r#"
         export default {
-            \u3131: '\u11B0',
+            "\u3131": '\u11B0',
```

Comment:
Nit: This is actually fine, `\u` escapes are [legal](https://tc39.es/ecma262/#prod-IdentifierName). It's just `\x` escapes that aren't.